### PR TITLE
Support for banning players

### PR DESCRIPTION
**Pull request by nonperforming**
Need to check position of ban button is okay or not!

Is it okay to save Jaket files to `Application.persistentDataPath`? Should it be saved where Jaket DLL is?

New dependency: Newtonsoft.JSON from `ULTRAKILL\ULTRAKILL_Data\Managed\Newtonsoft.JSON`

### CODE REVIEW

File: net/LobbyController.cs [hunk 1]
```diff
@@ -150,6 +157,28 @@ public static void KickMember(Friend member)
         Lobby?.SendChatString($"<system><color=red>Player {member.Name} was kicked!</color>");
     }
 
+    /// <summary> Bans the member from the lobby, or rather asks him to leave, because Valve has not added such functionality to its API. </summary>
+    public static void BanMember(Friend member)
+    {
+      // who does the client think he is?!
+      if (!IsOwner) return;
+
+      BannedPlayers.Add(member.Id);
```
Comment (nonperforming) [inline net/LobbyController.cs:166]: BannedPlayers should have content loaded from disk when player joins but this seems like a bad idea anyawys

File: net/LobbyController.cs [hunk 2]
```diff
@@ -3,12 +3,14 @@ namespace Jaket.Net;
 using Steamworks;
 using Steamworks.Data;
 using System;
+using Newtonsoft.Json;
```
Comment (nonperforming) [inline net/LobbyController.cs:6]: New dependency: From `ULTRAKILL\ULTRAKILL_Data\Managed\Newtonsoft.JSON`

**Comment by xzxADIxzx** [comment]
Ugh, I don't know if the mod needs a ban list 'cause there are no long-lived servers, there are only lobbies that don't last longer than a game session

**Comment by nonperforming** [comment]
> Ugh, I don't know if the mod needs a ban list 'cause there are no long-lived servers, there are only lobbies that don't last longer than a game session

Trolls/cheaters repeatedly (attempting to) lag or crash the game, or generally Not Pleasant™ people joining. Lobby hosts will need to keep checking the player list and kick cheaters/trolls which can get repetitive, (further insinuated as users can change their Steam name instantly (SteamID is not shown, but suppose this is a different issue))
Additionally, players currently can bypass the "recently kicked" check by either having an alternate account/friend and alternating between the two or simply waiting for somebody else to be kicked.

> And why didn't you use Unity's Json?

I didn't know that existed, but it appears we will need a `System.Serializable` class to store the SteamID if we are to use it if I am not mistaken

**Comment by xzxADIxzx** [comment]
1. Lags and crashes will be fixed, I just forgot to add a condition
2. Alt accounts cannot be fixed even with a ban system

**Pull request closed without merging**